src/transformers/modeling_utils.py

```diff
@@ -279,6 +279,16 @@ def _wrapper(*args, **kwargs):
     return _wrapper
 
 
+def get_torch_device_context_manager():
+    """
+    Test if a device context manager is currently in use.
+    """
+    test = torch.zeros(1)
+    if test.device == torch.get_default_device():
+        return None
+    return test.device
```

Comment:
not a big deal, but an edge case (which is likely no one does it): when we are in a context manager that is specified with the torch's default device, here it returns `None` but in theory it should return that default device.

But it seems these 2 don't make any difference ..?